nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml

```diff
@@ -0,0 +1,10 @@
+# Requires the Asset component and the Twig bundle
+app.swagger_ui:
+    path: /api/doc
+    methods: GET
+    defaults: { _controller: nelmio_api_doc.controller.swagger_ui }
+
+# app.swagger:
```

Comment:
i think this should have a comment above this line what it does

Reply:
added :)